gradle/openssl.gradle

```diff
@@ -0,0 +1,7 @@
+test { 
+    if (org.gradle.internal.os.OperatingSystem.current().isMacOsX()) {
+        def opensslDir = "/usr/local/opt/openssl/lib"
```

Comment:
Can we append this to `LD_LIBRARY_PATH` so that if one doesn't have openssl in `/usr/local/opt/openssl/lib` we can still make it work?